test/storage/local_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule ArcTest.Storage.Local do
+  use ExUnit.Case
+  @img "test/support/image.png"
+
+  setup_all do
+    File.mkdir_p("arctest/uploads")
+  end
+
+  defmodule DummyDefinition do
+    use Arc.Definition.Storage
+    @acl :public_read
+    def transform(:thumb, _), do: {:convert, "-strip -thumbnail 10x10"}
+    def __versions, do: [:original, :thumb]
```

Comment:
Can you add a custom filename function to this test case?  Right now you have two versions specified but they both will use the same destination filename of `"thumb.png"`.

```elixir
def filename(version,  {file, _}), do: "#{file.file_name}-#{version}"
```

And then check to make sure *both* the original **and** the thumbnail are saved correctly?